catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/bower.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "registry-admin-remote-ui",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "backbone.modelbinder": "v1.0.5",
+    "components-backbone": "1.1.0",
+    "backbone-poller": "0.2.8",
+    "components-bootstrap": "3.1.1",
+    "components-font-awesome": "4.0.3",
+    "icanhandlebarz": "atomicobject/ICanHandlebarz.js#f6ac1d1f8e1d796045563f1483eca6c4911ed7d1",
+    "handlebars": "2.0.0-alpha.2",
+    "html5shiv": "3.7.2",
+    "jquery": "components/jquery#1.11.0",
+    "jquery-ui": "1.10.4",
+    "marionette": "1.8.8",
+    "perfect-scrollbar": "0.4.8",
+    "q": "v1.0.1",
+    "requirejs": "2.1.14",
+    "requirejs-plugins": "v1.0.2",
+    "require-css": "0.1.5",
+    "bootswatch": "=v3.1.1",
+    "jquery-file-upload": "9.5.7",
```

Comment:
No, but I tried removing that line (and a few others I was sure I wasn't using) and my browser console started complaining... though the ui still seemed to work fine.